core/api/monitor/v1/monitor_proto/monitor.proto

```diff
@@ -28,13 +28,16 @@ import "trillian.proto";
 
 // GetMonitoringRequest contains the input parameters of the GetMonitoring APIs.
 message GetMonitoringRequest {
-  // epoch specifies the for which the monitoring results will
-  // be returned (epochs start at 1).
-  int64 epoch = 1;
-
-  // kt_URL is the URL of the keytransparency server for which the monitoring
+  // kt_url is the URL of the keytransparency server for which the monitoring
```

Comment:
Not totally convinced this commit belongs in this PR?  It seems like tangential work.